src/v/config/configuration.h

```diff
@@ -663,6 +663,7 @@ struct configuration final : public config_store {
     bounded_property<std::chrono::milliseconds> cpu_profiler_sample_period_ms;
     property<std::filesystem::path> rpk_path;
     property<std::optional<std::filesystem::path>> debug_bundle_storage_dir;
+    property<std::optional<uint32_t>> debug_bundle_auto_removal_seconds;
```

Comment:
Why not `std::chrono::seconds`?

nit: The variable then wouldn't need the units, but:
1. It wouldn't make much sense (should it be renamed?)
2. It would highlight that the config extraction tool uses the variable name, not the property::name